src/main.py

```diff
@@ -122,7 +122,7 @@ def update_message_dir(self) -> List[Message]:
     def upload_messages(self, messages_to_upload : List[str]):
         failed = 0
         for message_id in messages_to_upload:
-            email_thread : str = None
+            email_thread : Message
```

Comment:
oh, I'm surprised/sad mypy didn't catch this before?
We did annotate that `parse_comments()` takes a `Message`.

Reply:
I think Mypy did catch it but I didn't test it when I made the change because it was in response to a comment I think? IDK it was definitely just my mistake in not checking.

Reply:
Ack, sg.
I guess that means we can be a bit more confident in the code then if mypy isn't complaining anymore.

And no worries. We really should have setup an automated check w/ mypy so you wouldn't have to remember.
(And ideally, the type annotations would have been correct before you started working on it, but that wasn't the case either :P)